Makefile

```diff
@@ -50,30 +50,49 @@ endif
 endif
 export BASEIMAGE
 
+ifeq (${IMAGE_REPO}, )
+  IMAGE_REPO = "openebs"
```

Comment:
Why is the space required after the variable?  It would be good to follow a consistent way of setting the variable in the  Makefile.